.sass-lint.yml

```diff
@@ -28,7 +28,7 @@ rules:
     -
       include: true
 
-  nesting-depth: 2
+  nesting-depth: 3
```

Comment:
It's weird but the 2 here isn't actually the maximum nesting depth, it specifies that a failed nesting-depth check is an error (they use 1 for warning and 2 for error).

For an example of changing the options for a rule look at the config for indentation above. The options for nesting-depth are documented here: https://github.com/sasstools/sass-lint/blob/develop/docs/rules/nesting-depth.md

Reply:
Thanks for the quick tutorial. That's what I get for guessing! A classic example RTFM.